src/coreclr/src/tools/ReadyToRun.SuperIlc/CommandLineOptions.cs

```diff
@@ -232,6 +233,9 @@ public static CommandLineBuilder Build()
             Option InputFileSearchString() =>
                 new Option(new[] { "--input-file-search-string", "-input-file" }, "Search string for input files in the input directory", new Argument<string>());
 
+            Option GCStress() =>
+                new Option(new[] { "--gcstress" }, "Run tests with the specified GC stress level enabled (the argument value is in hex)", new Argument<string>());
```

Comment:
Nit - I wonder whether it might make sense to enumerate the allowed GC stress levels in the help string, I believe there are just 3 or so unless I'm mistaken.

Reply:
The GC stress level is a hex number with any bit combination of 5 bits, see:
```c++
    enum  GCStressFlags {
        GCSTRESS_NONE               = 0,
        GCSTRESS_ALLOC              = 1,    // GC on all allocs and 'easy' places
        GCSTRESS_TRANSITION         = 2,    // GC on transitions to preemtive GC
        GCSTRESS_INSTR_JIT          = 4,    // GC on every allowable JITed instr
        GCSTRESS_INSTR_NGEN         = 8,    // GC on every allowable NGEN instr
        GCSTRESS_UNIQUE             = 16,   // GC only on a unique stack trace
    };
```

Reply:
OK, my bad, I didn't know it was that complex. In such case just ignore my comment.